src/components/monitoring/__tests__/LiveMetricsDashboard.test.tsx

```
@@ -0,0 +1,134 @@
+        fps: 55,
+        networkLatency: 20
+      }
+    };
+
+    // Get the handler that was registered with subscribe
+    const mockCalls = (mockEventManager.subscribe as any).mock.calls;
+    const metricsCall = mockCalls.find(
+      (call: any[]) => call[0] === 'metrics.snapshot_collected'
+    );
+    
+    if (metricsCall && metricsCall[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)